### DISCUSSION TITLE
TheHive 4 upgrade

### DISCUSSION THREAD

**Discussion by idolaman** [Security-Onion-Solutions/securityonion#5729] [Unsupported Versions] [answered]
Hello.
There are plans to upgrade the TheHive version to 4 in the future?

Thanks :)

**Comment by clairmont32** [comment]
It's not listed as a task/issue in the Project boards. It doesn't mean it wont necessarily happen but it doesnt appear to be on the current "roadmap" by the boards.

**Comment by dougburks** [comment, marked as answer]
We have no plans to upgrade to TheHive 4. We will be changing how we do case management in the future.
